yarn-project/noir-contracts.js/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@aztec/aztec.js": "workspace:^",
+    "@aztec/noir-compiler": "workspace:^",
```

Comment:
Can we move this to a `devDependency`? Not a big deal though as we're not planning in publishing this package.